src/emulator/controller.c

```diff
@@ -149,7 +163,7 @@ bool fn_800622B8(Controller* pController) {
         pController->unk_BC[i] = pController->unk_CC[i] = 0;
         pController->stickLeft[i][AXIS_X] = pController->stickLeft[i][AXIS_Y] = 0;
         pController->stickRight[i][AXIS_Y] = 0;
-        pController->unk_21C = -1; // why here
+        pController->iString = ERROR_NONE; // why here
```

Comment:
I only just noticed this, reordering these statements doesn't work? Maybe with `pController->stickRight[i][AXIS_X] = pController->stickRight[i][AXIS_Y] = 0;`

Reply:
apparently this function isn't matched on main, I thought it was 🤔
it's hard to see if there's any improvement